dts/x86/quark_se_c1000_devboard.dts

```diff
@@ -0,0 +1,32 @@
+/dts-v1/;
+
+#include "intel_curie.dtsi"
+
+/ {
+	model = "quark_se_c1000_devboard";
+	compatible = "intel,quark_se_c1000_devboard", "intel,quark_se_c1000";
+
+	aliases {
+		uart_0 = &uart0;
+		uart_1 = &uart1;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart1;
+		zephyr,bt-uart = &uart0;
+		zephyr,uart-pipe = &uart1;
+		zephyr,bt-mon-uart = &uart1;
```

Comment:
what about serial over USB? did you test that with the board, how can I configure the serial console to go over CDC ACM? See related sample for serial usb in samples/subsys/usb